src/amo/utils/server.js

```diff
@@ -9,14 +10,21 @@ import log from 'amo/logger';
 
 const PREFIX = 'enableFeature';
 
-type ExpressHandler = (req: typeof $Request, res: typeof $Response) => void;
+type ExpressHandler = (
+  req: typeof $Request,
+  res: typeof $Response,
+) => void | Promise<Function>;
```

Comment:
I had to do this to deal with the `async` in `viewHeartbeatHandler`. I'm not sure if there is a better way to make this work with `Flow`.

Reply:
Shouldn't that be?

```suggestion
) => void | Promise<void>;
```